### PR TITLE
[Wait for #2102][Application] Add Simple Layer Tensorflow Examples

### PR DESCRIPTION
**Pull request by DonghakPark**
Add Simple Tensorflow Examples with Dummy Data
- Linear
- Conv
- LSTM
- Model_A_Linear
- Model_A_Conv
- Model_C_Linear
- Model_C_Conv

It has same Layer with NNtrainer example that has same name

Signed-off-by: DongHak Park <donghak.park@samsung.com>

### CODE REVIEW

File: Applications/Layers/Tensorflow/Conv.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+print("Tensorflow Verison : ", tf.__version__)
+print("Keras Version      : ", tf.keras.__version__)
+
+IMG_SIZE = 224
+OUTPUT_SIZE = 10
+
+
+class Model(tf.Module):
+    def __init__(self):
+
```
Comment (jihochu) [inline Applications/Layers/Tensorflow/Conv.py:23]: Is this empty line is PEP style?
please check: https://peps.python.org/pep-0008/#blank-lines

File: Applications/Layers/Tensorflow/Model_A_Conv.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2023 DongHak Park <donghak.park@samsung.com>
+#
+# @file   main.cpp
+# @date   01 Feb 2023
+# @see    https://github.com/nnstreamer/nntrainer
+# @author Donghak Park <donghak.park@samsung.com>
+# @bug	  No known bugs except for NYI items
+# @brief  This is Model_A_Conv Example for Tensorflow (with Dummy Dataset)
```
Comment (jihochu) [inline Applications/Layers/Tensorflow/Model_A_Conv.py:9]: what is Model_A_Conv ?

File: Applications/Layers/Tensorflow/Conv.py (hunk 1 above)
Comment (DonghakPark) [reply Applications/Layers/Tensorflow/Conv.py:22]: [Fix] Thank you for Review!!

File: Applications/Layers/Tensorflow/Model_A_Conv.py (hunk 2 above)
Comment (DonghakPark) [reply Applications/Layers/Tensorflow/Model_A_Conv.py:9]: [Fix] Add explain in brief

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by baek2sm — APPROVED** [review]
LGTM

**Pull request merged**